src/test/java/com/iota/iri/crypto/ISSTest.java

```diff
@@ -78,6 +78,28 @@ public void addressGenerationISS() throws Exception {
         }
     }
 
+    @Test
+    public void sigDigestISSInPlace() throws Exception {
+        int index = 0;
+        int nof = 1;
+        SpongeFactory.Mode[] modes = {SpongeFactory.Mode.KERL};
+        Hash[] hashes = {HashFactory.ADDRESS.create("DQMAVYJTIIQUCDHRIZAPYKFCFBTHNDAMHLYEEIEBSR9REQBBVUQOSDWFSGPTZFNQQTVMQPVUDHDTEZVSW")};
```

Comment:
Can you split it to two lines after the `=`?
Our static code analysis will pick this up later on

Reply:
not sure I understand the reason, the static code analysis will think it is too wide?